amplify/backend/function/questionslist/src/index.js

```diff
@@ -0,0 +1,14 @@
+const { parseQuestions, batchWriteQuestions } = require('./questions');
+
+const MAX_ITEMS_PER_DB_WRITE = 25;
```

Comment:
Perhaps putting this in the config file or env file would be better since we might want to be consistent across all db writes for this lambda function (if there ends up being more than one type of write call being made)? The scope of this is only for the lambda function tho so I think either way is fine.

Reply:
Think it's probably better to keep it here since it's only being used here and isn't "config" per-say- just a constant that needs to be used only at the top-level